app/views/hyrax/dashboard/collections/_default_group.html.erb

```diff
@@ -0,0 +1,30 @@
+<table id="collections-list-table" class="table table-striped collections-list-table">
```

Comment:
This file doesn't have any tests.  Could you please add some or is it effectively covered elsewhere?